core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2681,6 +2682,9 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
     getPermissionsFromRoles(googleRoles)
   }
 
+  private def getStatusCodeHandlingUnknown(intCode: Integer) =
```

Comment:
I decided that while I'm mucking some more with this error handling, might as well fix this issue too.

https://broad-institute.sentry.io/issues/4044258270/?project=153297&query=is%3Aunresolved&referrer=issue-stream&stream_index=12